### ISSUE TITLE
Update check dialog can't update

### ISSUE THREAD

**Issue by hannesa2** [gitx/gitx#299]
I use the arm64 version and I see this
<img width="845" alt="image" src="https://user-images.githubusercontent.com/3314607/157099522-739f3af8-e404-43bd-8b7a-bf49873af779.png">

I can "Learn more..." , "Skip this version" or "Remind me later" but my expectation is an "Update" button.
Is my exception wrong or is our logic wrong ?

**Comment by hannesa2** [comment]
I see there is theoretically such a button https://sparkle-project.org/

<img width="614" alt="image" src="https://user-images.githubusercontent.com/3314607/157100675-6ff8f06d-0dcb-4549-9045-31eb0212ced9.png">

**Comment by claytonrcarter** [comment]
Yes, it's totally possible, but we disabled it in #293 (947202ac8232b9e9b0c936006110e743b11a1fb2) b/c we aren't/can't sign the DMGs, so the "Install Update" button was generating an error. 

Also, as I recall, Sparkle expects us to bundle universal apps, not independent builds like we're currently doing. So there's a number of reason's why that wasn't working, I guess.

If we can build a universal app, and signed the dmgs, then we can rework the sparkle XML and go back to "Install Update", but that's a lot of "ifs". 😄

**Comment by radiospiel** [comment]
> I can "Learn more..." , "Skip this version" or "Remind me later" but my expectation is an "Update" button.
Is my exception wrong or is our logic wrong ?

**Comment by hannesa2** [comment]
I guess even when you installed it with `brew`, it will not work
